### PR TITLE
Added a question to general-html-quiz.ts

### PR DESCRIPTION
**Pull request by Harshit-Raj-14**
Added a question based on the application of href tag in an html document code.

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

Please add a **hacktoberfest accepted** tag after merged.

Thanks for the opportunity.

### CODE REVIEW

File: src/data/html-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,16 @@
 const htmlQuiz = [
+  {
+    Question:
+    "Which of the following is the correct way of creating an hyperlink in HTML?",
```
Comment (jdwilkin4) [inline src/data/html-quiz.ts:4]:
```suggestion
    "Which of the following is the correct way of creating an anchor element in HTML?",
```

File: src/data/html-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,16 @@
...
+    Answer: "<a href= “www.freecodecamp.org”> freeCodeCamp </a>",
+    Distractor1: "<a>www.freecodecamp.org <freeCodeCamp /a>",
+    Distractor2: "<a href=“ www.freecodecamp.org” freeCodeCamp /a>",
+    Distractor3: "<a link=“ www.freecodecamp.org” freeCodeCamp > </a>",
```
Comment (jdwilkin4) [inline src/data/html-quiz.ts:8]: let's just go with single quotes here

```suggestion
    Answer: "<a href= 'www.freecodecamp.org'> freeCodeCamp </a>",
    Distractor1: "<a>www.freecodecamp.org <freeCodeCamp /a>",
    Distractor2: "<a href='www.freecodecamp.org' freeCodeCamp /a>",
    Distractor3: "<a link='www.freecodecamp.org' freeCodeCamp > </a>",
```

File: src/data/html-quiz.ts [hunk 3]
```diff
@@ -1,4 +1,16 @@
...
+    Explanation:
+      "In HTML a href tag is used to hyperlink text as shown in the given syntax. <a href= “www.link_name.com”> hyperlinked_text </a>"
```
Comment (jdwilkin4) [inline src/data/html-quiz.ts:10]:
```suggestion
      "In HTML, an anchor element is used to create links on a a web page. Here is the correct syntax: <a href= 'www.freecodecamp.org'> freeCodeCamp </a>"
```

File: src/data/html-quiz.ts [hunk 4]
```diff
@@ -1,4 +1,16 @@
...
+    Link:
+      "https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-href"
```
Comment (jdwilkin4) [inline src/data/html-quiz.ts:12]: Whenever possible, it is always best to use a freeCodeCamp resources first

```suggestion
      "https://www.freecodecamp.org/news/the-html-a-tag-anchor-tag-example-code/"
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/html-quiz.ts (hunk 3 above)
Comment (jdwilkin4) [reply src/data/html-quiz.ts:10]: Also, this should fix your syntax issue

**Comment by Harshit-Raj-14** [comment]
> Thank you for your contribution!
> 
> Please review the requested changes. Once those changes have been implemented, then we can merge your PR into main. 👍

Thank you, and I have made the necessary changes you have suggested.
Hopefully now the PR can be merged.

Also, please add a **hacktoberfest accepted** tag after merged.

Thanks for the opportunity.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by jdwilkin4** [comment]
Also, I resolved your other syntax error.
In the future, make sure to pay close attention to syntax errors 👍

**Pull request merged**